File: exercicios_C/1.exercSequenciais/exerc14.c
#include <stdio.h>

void main(){
    int peso_exc = 0;
    float peso, excesso, multa = 0;
    printf("informe a quant de peixe: KG ");
    scanf("%f", &peso);
    peso_exc = (int)(peso - 50);
    printf("peso excedente: KG %d.", peso_exc);
    printf("multa por peso excedente: R$%d.", peso_exc * 4);
}

File: exercpythonbrasil/1.exerc_sequencial/exerc14.py
"""
João Papo-de-Pescador, homem de bem, comprou um microcomputador para controlar o rendimento diário de seu trabalho. Toda vez que ele
traz um peso de peixes maior que o estabelecido pelo regulamento de pesca do estado de São Paulo (50 quilos) deve pagar uma multa de
R$ 4,00 por quilo excedente. João precisa que você faça um programa que leia a variável peso (peso de peixes) e calcule o excesso.
Gravar na variável excesso a quantidade de quilos além do limite e na variável multa o valor da multa que João deverá pagar. Imprima
os dados do programa com as mensagens adequadas.
"""
peso = int(input("Peso de peixes: "))
excesso = peso - 50
multa = excesso * 4
if peso <= 50:
    print("Você está dentro do regulamento!")
else:
    print("Você deverá pagar:",multa,"R$")
